generation/generate_release_please_config.sh

```diff
@@ -1,70 +1,60 @@
 #!/bin/bash
 
-set -xe
+set -e
 
-echo "{" > .release-please-manifest.json
+GENERATION_DIR=$(dirname -- "$0")
 
-echo ""\""google-cloud-gapic-bom"\"": "\""0.0.0"\""," >> .release-please-manifest.json
+# Hardcode the tab as two spaces instead of \t
+tab="  "
+release_please_manifest_file=".release-please-manifest.json"
+release_please_config_file="release-please-config.json"
 
-GENERATION_DIR=$(dirname -- "$0");
+[ -e "${release_please_manifest_file}" ] && rm "${release_please_manifest_file}"
+[ -e "${release_please_config_file}" ] && rm "${release_please_config_file}"
 
-num_modules=$(find . -mindepth 2 -maxdepth 2 -name pom.xml | wc -l)
-
-rp_config_line=""
-rp_manifest_line=""
+echo "{" > "${release_please_manifest_file}"
 
-for module in $(find . -mindepth 2 -maxdepth 2 -name pom.xml |sort | xargs dirname); do
-
-  if [[ "${module}" = *google-cloud-gapic-bom ]] || [[ "${module}" = *CoverageAggregator ]]; then
+# We are manually excluding the CoverageAggregator module -- TODO: Update this for any other exclusions
+num_modules=$(find . -mindepth 2 -maxdepth 2 -name pom.xml | wc -l)
+num_modules=$((num_modules - 1))
+for path in $(find . -mindepth 2 -maxdepth 2 -name pom.xml | sort | xargs dirname); do
```

Comment:
Can you add "--dictionary-order" parameter to `sort`?

https://github.com/googleapis/google-cloud-java/issues/8520